source/main/modules/window-manager/index.ts

```diff
@@ -219,6 +235,64 @@ export default class WindowManager {
       return
     }
 
+    this._mainWindow.on('show', () => {
+      const leaveAppRunning = Boolean(global.config.get('system.leaveAppRunning'))
+      if (leaveAppRunning) {
+        if (process.platform === 'darwin') {
+          if (this._tray == null) {
+            let basepath = app.getAppPath()
```

Comment:
Variable `basepath` is not used. I suggest remove this line.

Reply:
will remove in next commit